modules/ROOT/pages/validate-4-task.adoc

```diff
@@ -1,19 +1,34 @@
-= To Validate Query Parameters and Headers
+= Validate Query Parameters and Headers Using APIKit
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-partial:
 
-This procedure assumes that you have generated Mule flows using APIkit in Studio. You can configure APIkit Router to validate, or not, headers and query parameters. Strict query parameter and header validation limits parameters to those defined in the RAML. Enabling validations can affect performance. Many complex validations can cause a delay in response time, so if your use case cannot tolerate latency, you might want to disable validations. By default validations are disabled.
+This process assumes that you have generated flows using APIkit in Studio.
 
-. In Studio, select APIkit Router.
+You can configure APIkit Router to validate headers and query parameters to limit parameters to those defined in your API specification:
```

Comment:
The meaning of the sentence seems ambiguous. Is "query" being used as a noun or as a verb? In other words, is the meaning A or B?
A. You can configure APIkit Router to validate _headers and query parameters_ to limit parameters to...
B. You can configure APIkit Router _to validate headers_ and _to query parameters_ to limit parameters to...
Also, in a sentence further down, you say that validation also limits headers to those defined in an API spec. Why is that not said here, too?